### PR TITLE
chromium: update to 96.0.4664.110

### PR DESCRIPTION
**Pull request by rwmacleod**
https://chromereleases.googleblog.com/2021/12/stable-channel-update-for-desktop_13.html

The patch, musl/chromium-95-quiche-include.patch was upstreamed by:
   https://quiche.googlesource.com/quiche/+/cde409004ec894d7651a73ba1986ad2efe3683b5%5E%21/#F0
so drop it.

Test-built and run for:
  * chromium-x11
    * master,  clang, MACHINE="qemux86-64,qemuarm64"
    * master,  clang, musl, MACHINE="qemux86-64"

Test-built for:
* chromium-ozone-wayland:
  * honister,  clang, MACHINE="intel-corei7-64"
  * honister,  clang, MACHINE="raspberrypi3
  * hardknott, clang, MACHINE="raspberrypi3
  * dunfell,   clang, MACHINE="qemuarm64"
  * dunfell,   clang, MACHINE="m3ulcb"
* chromium-x11
  * honister,  clang, MACHINE="raspberrypi3"
  * hardknott, clang, MACHINE="raspberrypi3"
  * dunfell,   clang, MACHINE="raspberrypi3"

Signed-off-by: Randy MacLeod <Randy.MacLeod@windriver.com>

### CODE REVIEW

File: meta-chromium/recipes-browser/chromium/chromium.inc [hunk 1]
```diff
@@ -4,8 +4,8 @@ HOMEPAGE = "https://www.chromium.org/Home"
 CVE_PRODUCT = "chromium:chromium google:chrome"
 
 SRC_URI = "https://commondatastorage.googleapis.com/chromium-browser-official/chromium-${PV}.tar.xz"
-SRC_URI[md5sum] = "42de983018667ed4233460c040321c81"
-SRC_URI[sha256sum] = "488c6ad983ebf7781cb4d704f70496e8aa2165611b46656d7aa62f269c760407"
+SRC_URI[md5sum] = "e1243091e4b2a58467a01122cc240c9a"
```
Comment (kraj) [inline meta-chromium/recipes-browser/chromium/chromium.inc:7]: we dont need md5sums anymore, it can be dropped.

Comment (rwmacleod) [reply meta-chromium/recipes-browser/chromium/chromium.inc]: Done.

**Comment by msisov** [comment]
Our bots completed testing this build. Please add the following test targets to the commit message -

Test-built with:
* chromium-ozone-wayland:
  * honister,  clang, MACHINE="intel-corei7-64"
  * honister,  clang, MACHINE="raspberrypi3
  * hardknott, clang, MACHINE="raspberrypi3
  * dunfell,   clang, MACHINE="qemuarm64"
  * dunfell,   clang, MACHINE="m3ulcb"

* chromium-x11
  * honister,  clang, MACHINE="raspberrypi3"
  * hardknott, clang, MACHINE="raspberrypi3"
  * dunfell,   clang, MACHINE="raspberrypi3"

**Review by rakuco — APPROVED** [review]
Thanks everyone for testing and working on this.

@rwmacleod as @msisov said, please edit the commit message of your first commit and add the additional platforms that have been tested. I've updated the PR message, but I can't edit your commit.

For future PRs, please remember you can just rebase your commits and edit them, and that's preferred over closing PRs and sending new ones (e.g. #595 vs #596). We can help you with that if needed.

**Comment by rwmacleod** [comment]
Done, via:
$ git push --force: 
 + aa9bd4c...3527e4e update-to-96.0.4664.110-v2 -> update-to-96.0.4664.110-v2 (forced update)

Regarding using a new PR, are you saying that I should use git push --force like I have just now?

**Comment by rakuco** [comment]
> Regarding using a new PR, are you saying that I should use git push --force like I have just now?

Yep, exactly. Thanks for adjusting the commit message.

**Pull request merged**